Recast/Include/Recast.h

```diff
@@ -822,81 +825,110 @@ void rcClearUnwalkableTriangles(rcContext* ctx, const float walkableSlopeAngle,
 								const int* tris, int nt, unsigned char* areas); 
 
 /// Adds a span to the specified heightfield.
+/// 
+/// The span addition can be set to favor flags. If the span is merged to
+/// another span and the new @p smax is within @p flagMergeThr units
```

Comment:
Old parameter names are still used here and below: smax, flagMergeThr, Limit: < @p smax

Reply:
Fixed in 86383fc